pages/query-api/read-queries.mdx

```diff
@@ -129,10 +129,10 @@ Enables skipping over a specified number of documents, useful for implementing p
 
 `startAt`, `startAfter`, `endAt`, and `endAfter` are supported.
 
-- `startAt` : Begin fetching documents starting at a specific value.
-- `startAfter` : Fetch documents after a specific value.
+- `startAt` : Start fetching documents from a specific value.
+- `startAfter` : Fetch documents beginning immediately after a specific value.
 - `endAt` : Fetch documents up to a specific value.
-- `endAfter` : Fetch documents until just after a specific value.
+- `endAfter` : Fetch documents up to and including a specific value.
```

Comment:
btw does `endAt` include the specific value? and does `endAfter` include the value after the specified one?

if no, keep it as it is. If yes, please change to:

- `endAt`: Fetch documents up to and including a specific value.
- `endAfter`: Fetch documents up to, including, and the one after a specific value.